modules/administration_manual/pages/configuration/server/occ_app_command.adoc

```diff
@@ -189,7 +189,7 @@ Parameters for short form options will be written either directly after the opti
 sudo -u www-data ./occ files:scan -p "user_x/files/folder"  
 ....
 
-[[brute-force-protection]]
+[[ocapps_brute-force-protection]]
```

Comment:
Why don't you just remove these, and use [automatic anchors](https://asciidoctor.org/docs/user-manual/#automatic-anchors) instead?

Reply:
What's the reason for not using automatic anchors?